src/Akka.Persistence.SqlServer/Journal/SqlServerJournal.cs

```diff
@@ -64,5 +69,21 @@ protected override DbConnection CreateDbConnection(string connectionString)
         {
             return new SqlConnection(connectionString);
         }
+
+        /// <inheritdoc />
+        protected override void PreStart()
+        {
+            base.PreStart();
+
+            // if constant parameter sizes required, provide column sizes to query executor
+            if (_useConstantParameterSize)
+            {
+                using (var connection = CreateDbConnection(GetConnectionString()))
+                {
+                    var columnSizes = ColumnSizeLoader.LoadJournalColumnSizes(QueryExecutor.Configuration, connection);
+                    (QueryExecutor as SqlServerQueryExecutor)?.SetColumnSizes(columnSizes);
```

Comment:
Passing in the column sizes, so that `QueryExecutor` could use it for query generation (it's this guy who is generating parameters for db query).